dev-packages/e2e-tests/test-applications/nuxt-3-min/package.json

```diff
@@ -17,14 +17,23 @@
   },
   "dependencies": {
     "@sentry/nuxt": "latest || *",
-    "nuxt": "3.7.0"
+    "@sentry/core": "latest || *",
```

Comment:
This is only installed because semantic attributes are imported from `@sentry/core` in the tests -> could it be that those are also exported from `@sentry/nuxt`?